oauth2_http/pom.xml

```diff
@@ -91,6 +91,17 @@
       <artifactId>junit</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.google.truth</groupId>
+      <artifactId>truth</artifactId>
+      <version>1.0.1</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.google.code.gson</groupId>
+      <artifactId>gson</artifactId>
+      <version>2.8.6</version>
```

Comment:
Looks like this is only used in test: we should add test scope here.

Also, we're already using Jackson in other places (for better or worse), so maybe we don't need 2 json libraries. We can switch to gson across the board separately.

Reply:
Ok, removed